students/lakatorz_izaak/lesson_03_functions/the_collatz_sequence.py

```diff
@@ -0,0 +1,22 @@
+# Write a function named collatz() that has one parameter named number. If
+# number is even, then collatz() should print number // 2 and return this
+# value. If number is odd, then should print and return 3 * number + 1.
+
+
+def collatz(number):
+    if number % 2 == 0:
+        result = number // 2
+    else:
+        result = 3 * number + 1
+    print(str(result))
+
+    if result == 1:
+        return 0
```

Comment:
Uwaga na marginesie, bo jak poprawisz program to będzie nieadekwatna: nie wiem czy taki był cel, ale zrobiłeś tutaj funkcję, która albo zwraca 0 albo zwraca None.
Tutaj zwraca 0